docs/orchestration.asciidoc

```diff
@@ -0,0 +1,104 @@
+[id="{p}-orchestration"]
+=== NodeSets orchestration
```

Comment:
What do we think about calling this just `Orchestration` and mentioning nodesets later? We are talking about nodesets config, but also about stateful sets, behaviors and (as we talked offline) updateStrategy. Mentioning NodeSets feels not broad enough to me.

Reply:
I can do that. Just "Orchestration" may not be very explicit though. We could also go with `Nodes orchestration`, `Cluster upgrades and orchestration`?

Reply:
👍 to remove `NodeSets` , I would be in favor of `Nodes Orchestration`

Reply:
+1 for Nodes Orchestration